src/render/extract.rs

```diff
@@ -75,52 +75,52 @@ pub fn vector_instances(
     }
 }
 
-#[derive(Component, Clone)]
+#[derive(Component)]
 pub struct ExtractedRenderText {
-    pub font: Handle<VelloFont>,
+    pub font: VelloFont,
```

Comment:
Is Handle<T> gone in 0.13? How do we get the asset then? I want to make sure this isn't causing unnecessary cloning every frame (bad performance)